main.go

```diff
@@ -66,6 +69,15 @@ func main() {
 		exit(1)
 	}
 
+	hasher := sha256.New()
+	s, err := ioutil.ReadFile(path)
+	hasher.Write(s)
+	if err != nil {
+		fmt.Fprintf(stderr, "Error calculating checksum for '%s': %s\n", path, err.Error())
+		exit(1)
+	}
+	zoneChecksum := hex.EncodeToString(hasher.Sum(nil))
```

Comment:
`hex.EncodeToString` is not needed. `fmt.Printf` can format properly using `%x`.

Reply:
done!